CrossprojectpipingExternalModule.php

```diff
@@ -953,7 +953,7 @@ function pipeToRecord($dst_rid) {
 					
 					// skip this event if the event_id isn't valid (eid is only valid if it has the same name as the name of the event contains the form that contains the destination match field)
 					# Quick-Fix for PHP8 Support
-					if (in_array($eid, (array) $source_project['valid_match_event_ids']) === false) {
+					if (in_array($eid, (array) $src_project['valid_match_event_ids']) === false) {
```

Comment:
Typo: `$source_project` is not defined and is null at this point.

Reply:
This changes looks great!  Resolving...